docs/topics/mta-rn-known-issues-7-0-3.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-3_{context}"]
+= Known issues
+
+{ProductFullName} version 7.0.3 has the following known issues.
+
+.IntelliJ IDE plugin Keymap actions do not work
+// Scheduled to be fixed in 7.0.3. Move this to resolved issues once fixed?
+
+In order to automate {ProductShortName}, the IntelliJ IDE plugin Keymap actions must work. They currently do not work.  link:https://issues.redhat.com/browse/MTA-2460[(MTA-2460)]
```

Comment:
Attempted to reword, not sure if technically correct. Also, would be great to add some more details. For example, what caused this issue.

```suggestion
The IntelliJ IDE plugin Keymap actions do not work which prevents the {ProductShortName} automation. link:https://issues.redhat.com/browse/MTA-2460[(MTA-2460)]
```